src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -56,6 +56,9 @@ public partial class HubConnectionContext
     [MemberNotNullWhen(true, nameof(_messageBuffer))]
     internal bool UsingStatefulReconnect() => _useStatefulReconnect;
 
+    // Tracks groups that the connection has been added to
+    internal HashSet<string> GroupNames { get; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
> `OrdinalIgnoreCase`

That's technically a breaking change. I realize the `RedisHublifetimeManager` uses ignore case as well, not sure why that is (might be an accident), but the default isn't ignore case. We shouldn't make that decision as part of this PR, so let's change to `Ordinal`.

Reply:
File an issue?